nectar/src/seed.rs

```diff
@@ -62,7 +62,7 @@ impl fmt::Debug for Seed {
 
 impl fmt::Display for Seed {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(f, "{:?}", self)
+        write!(f, "{:#}", self)
```

Comment:
This looks like it is an unintended change :)

Reply:
Thanks, sorry about that, I did try to check but I clearly missed one.